les_2/HW_2/task_hw_2_1.py

```diff
@@ -0,0 +1,25 @@
+# Напишите программу, которая получает целое
+# число и возвращает его шестнадцатеричное
+# строковое представление. Функцию hex
+# используйте для проверки своего результата.
+
+BASE = 16
+
+
+def convert(input_number: int) -> str:
+    output_number = []
+    translate_to_hex = {10:'a', 11:'b', 12:'c', 13:'d', 14:'e', 15:'f'}
+    while input_number > 0:
+        delimeter = input_number % BASE
+        for keys_for_hex in translate_to_hex:
+            if delimeter == keys_for_hex:
+                delimeter = translate_to_hex.get(keys_for_hex)
+        output_number.insert(0, str(delimeter))
```

Comment:
Тогда можно упросить
```suggestion
    while input_number:
        output_number.insert(0, translate_to_hex[input_number % BASE])
```